pkg/plugins/models.go

```diff
@@ -21,14 +21,29 @@ var (
 	PluginStateAlpha PluginState = "alpha"
 )
 
-type PluginSignature string
+type PluginSignatureState struct {
+	Status     PluginSignatureStatus
+	Type       PluginSignatureType
+	SigningOrg string
+}
+
+type PluginSignatureStatus string
+
+const (
+	PluginSignatureInternal PluginSignatureStatus = "internal" // core plugin, no signature
+	PluginSignatureValid    PluginSignatureStatus = "valid"    // signed and accurate MANIFEST
+	PluginSignatureInvalid  PluginSignatureStatus = "invalid"  // invalid signature
+	PluginSignatureModified PluginSignatureStatus = "modified" // valid signature, but content mismatch
+	PluginSignatureUnsigned PluginSignatureStatus = "unsigned" // no MANIFEST file
```

Comment:
Can you de-export these as they're only used within the package? If they are to be exported, the comments should be written as descriptive sentences, on account of being part of the package API.
```suggestion
	pluginSignatureInternal PluginSignatureStatus = "internal" // core plugin, no signature
	pluginSignatureValid    PluginSignatureStatus = "valid"    // signed and accurate MANIFEST
	pluginSignatureInvalid  PluginSignatureStatus = "invalid"  // invalid signature
	pluginSignatureModified PluginSignatureStatus = "modified" // valid signature, but content mismatch
	pluginSignatureUnsigned PluginSignatureStatus = "unsigned" // no MANIFEST file
```